src/main/java/Todo.java

```diff
@@ -0,0 +1,11 @@
+public class Todo extends Task{
+
+    public Todo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String descTask() {
```

Comment:
Maybe consider changing the name of the method to be more of a verb, such as describeTask.
However if the purpose if to print out information about the object, maybe using a toString method would be useful